tests/test_person.py

```diff
@@ -0,0 +1,31 @@
+from unittest import TestCase
+
+from src.models.person import Person
+from src.models.wikimedia.wikipedia.templates.enums import (
+    EnglishWikipediaTemplatePersonRole,
+    WikipediaTemplatePersonRole,
+)
+
+
+class TestPerson(TestCase):
+    def test_full_name(self):
+        p = Person(given="test", surname="test")
```

Comment:
Add a test where `name_string` = "   " and `given` = "   " and `surname` = "   "